core/src/main/resources/jenkins/model/experimentalflags/BooleanUserExperimentalFlag/flagConfig.jelly

```diff
@@ -0,0 +1,32 @@
+<!--
+The MIT License
+
+Copyright (c) 2022, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <select class="jenkins-select__input" name="[${it.flagKey}]">
```

Comment:
I think https://weekly.ci.jenkins.io/design-library/ToggleSwitch/ would be a more appropriate control

Reply:
The problem with the ToggleSwitch (AFAICT) is that it has only two states.
Having the "default" is important from my PoV so that we can let people decide to "go back" to default.
Huge inspiration from chrome://flags

![Screenshot_2022-10-28_181312_001](https://user-images.githubusercontent.com/2662497/198684080-19c71acb-fe6a-4a78-bcd3-b46e1f6d3be0.png)

Reply:
What is the default state? (i.e. does this need to be shown visually, in the description or should we simplify with just two states)

Reply:
The default value in the dropdown is "Default". It means that the user does accept the value provided in the code (`getDefaultValue`).

My idea is that we introduce a new feature, we start with disabled by default, so interested users can enable the behavior. During that period, the author can continue improving their feature (without having to wait too much). Then after some months, one can move the feature to be enabled by default. At that moment, the users could decide to rollback by using Disabled. 

In parallel, a user who decide that the feature is interesting they can enabled it in advance and when the feature default behavior will be changed, there is no impact for them.

With the different behavior we can see there, we can gather telemetry to determine if the feature is something the community wants.

I want to be able to see data about adoption instead of just having the opinion from loud users (often either very happy or very unhappy).

Reply:
I mean how does the user know what the default state is? 'Default' doesn't imply what the default actually is.

Reply:
oh, in that sense, got it! I will see what I can adjust 👍

Reply:
Tell me what you think :) 

![Screenshot_2022-11-04_121501_001](https://user-images.githubusercontent.com/2662497/199965607-7411aeda-e0e6-4875-9330-f74d9bd36ceb.png)

Reply:
Yes that's better

Reply:
Could the "(default)" be added next to the default entry instead of adding one just for that?

Reply:
I would say that putting the "help" the closer to where it is used seems better (yeah talking about some pixels :D)

Reply:
After a quick discussion with Hervé (due to my misunderstanding of his argument), I think I forgot to mention the most important argument about the tristate.
As I would like to introduce telemetry on the flags to ease the data gathering, having the "default" option is a way to have a "no vote" / "blank vote" in terms of adoption of the feature.

Reply:
Make perfect sense to have a third default state in that case indeed.